package.json

```diff
@@ -180,5 +182,16 @@
 			"<rootDir>/.cache/",
 			"<rootDir>/.public/"
 		]
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "npm test"
```

Comment:
Actually, no I don't think we should run tests before a commit. We have CI for that.

Reply:
I know, but it will help people like me :).

Reply:
> IF machine==jey npm test

Reply:
Fine, I'll try it. If it gets on my nerves, I'm ripping it out though :P